python/simulators/CMakeLists.txt

```diff
@@ -71,3 +60,32 @@ if(OPM_ENABLE_PYTHON_TESTS)
       -E env PYTHONPATH=${PYTHON_PATH} ${PYTHON_EXECUTABLE}
       -m unittest test/test_schedule.py)
 endif()
+
+set(_opm_common_python_script_dir "${opm-common_DIR}/python")
+if (NOT EXISTS "${_opm_common_python_script_dir}/install.py" )
+  # if the install.py script does not exist, assume that we are building against an installed opm-common
+  set(_opm_common_python_script_dir "${opm-common_PYTHON_COMMON_DIR}")
+endif()
+
+# NOTE: instead of using file( COPY ...) which copies the files at configure time (not at build time)
+#  we should add copying of the files at build time such that running "make" or "make all" will
+#  update the files if they have been modified. We use the install.py script in opm-common, see also
+#  CMakeLists.txt in opm-common
+add_custom_target(copy_python ALL
+    COMMAND ${PYTHON_EXECUTABLE} "${_opm_common_python_script_dir}/install.py"
+      ${PROJECT_SOURCE_DIR}/python ${PROJECT_BINARY_DIR} 0)
```

Comment:
Mhh, we already have problems on Debian's salsa-ci because our build artefacts are to big. With this change it seems we copy more to the binary directory than before (e.g. pybind). Is that really needed?

Reply:
Yes you are right, I believe the same thing is happening with opm-common, see [line 416](https://github.com/OPM/opm-common/blob/dbec8e4466b5b3293dd41067aaeb13566523bd34/CMakeLists.txt#L416) in `CMakeLists.txt`. I have submitted to new PRs, see  https://github.com/OPM/opm-common/pull/3043 and #3938  to address this.